guide/sequelize/currency.md

```diff
@@ -314,16 +314,16 @@ We use `.add()` for both removing and adding currency. Since we already check if
 
 ```js
 const item = await CurrencyShop.findOne({ where: { name: { $like: commandArgs } } });
```

Comment:
As mentioned in Discord, this should be `$iLike`. It seems like the code sample is already using `$iLike`, so just update it here and in the text below.